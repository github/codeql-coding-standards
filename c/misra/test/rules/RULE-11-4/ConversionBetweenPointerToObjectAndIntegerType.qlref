rules/RULE-11-4/ConversionBetweenPointerToObjectAndIntegerType.ql