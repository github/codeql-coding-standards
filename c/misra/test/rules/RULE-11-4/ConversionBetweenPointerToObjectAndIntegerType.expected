| test.c:5:21:5:42 | (unsigned int)... | Cast performed between a pointer to object type and a pointer to an integer type. |
| test.c:5:35:5:42 | (int *)... | Cast performed between a pointer to object type and a pointer to an integer type. |
| test.c:6:21:6:37 | (unsigned int)... | Cast performed between a pointer to object type and a pointer to an integer type. |
| test.c:8:8:8:24 | (unsigned int)... | Cast performed between a pointer to object type and a pointer to an integer type. |
| test.c:10:22:10:22 | (unsigned int *)... | Cast performed between a pointer to object type and a pointer to an integer type. |
| test.c:12:22:12:39 | (unsigned int *)... | Cast performed between a pointer to object type and a pointer to an integer type. |
